util/test/start_test.py

```diff
@@ -1465,6 +1468,9 @@ def parser_setup():
     parser.add_argument("-junit-remove-prefix", "--junit-remove-prefix",
             action="store", dest="junit_remove_prefix", metavar="<prefix>",
             help=help_all("<prefix> to remove from tests in jUnit report"))
+    # respect skipifs
+    parser.add_argument("-respect-skipifs", "--respect-skipifs",
+            action="store", dest="respect_skipifs")
```

Comment:
This doesn't work, because the action for a boolean flag in `argparse` must be `store_true` or `store_false`: https://docs.python.org/3/library/argparse.html#action

If you have `store`, then the flag expects an argument. Changing this to `store_true` made this PR work for me on few cases.